libs/langgraph/langgraph/pregel/draw.py

```diff
@@ -120,16 +125,16 @@ def draw_graph(
                                     edges.add((task.name, t[0], True, t[2]))
                             writes = [t for t in writes if t[0] != END]
                             conditionals.update(
-                                {(task.name, *t[:2]): t[2] for t in writes}
+                                {(task.name, t[0], t[1] or None): t[2] for t in writes}
```

Comment:
Why is this change needed?

Reply:
because the empty writes values in case of list/dict are unhashable

Reply:
(needed for the line below, and this one is just for consistency)